google/cloud/forseti/notifier/notifier.py

```diff
@@ -151,16 +150,20 @@ def process(message):
             payload.get('email_description'))
         return
 
-def main(_):
-    """Main function.
+# pylint: disable=unused-argument
+def run(forseti_config, inventory_index_id, service_config=None):
```

Comment:
The args here assume forseti_config is a dict. But from the cli to service and notifier it is all referred as config_file and in cli is referred to as a path. If the customer input "forseti notifier --config_file ./PATH/ config" it will cause problem.

Reply:
Dropping this.  This is now server-side flag.